packages/sources/react/src/components/indicators/VtmnRating/VtmnRating.tsx

```diff
@@ -105,7 +105,7 @@ export const VtmnRating = ({
         className,
       )}
       aria-disabled={disabled}
-      {...props}
+      {...(readonly && props)}
```

Comment:
2 questions:
- it is not clear to me why there is a condition on the readonly props?
- no tested but does this work? Since you are spreading, shouldn't you use a ternary here like `{...(readonly ? props : {})}` ?

Reply:
For the ternary, it is not necesary, 
<img width="282" alt="image" src="https://user-images.githubusercontent.com/2856778/233284384-13173f3a-b158-4d17-8547-8f538288c838.png">

The condition is on the readonly part because, if we are on readonly true / false, the structure are not the same. 

<img width="691" alt="image" src="https://user-images.githubusercontent.com/2856778/233285154-a4d5d1db-d3d7-4398-adb1-a0731b17a423.png">

With readonly false, we have a sub level with `vtmn-rating--interactive` which is contains all properties like aria-labels etc ... who the user can override with the restProps. 

This level doesn't exist if readonly is true.
<img width="533" alt="image" src="https://user-images.githubusercontent.com/2856778/233285273-7ea85dfc-2ce3-421d-ac4d-11026ec333e1.png">

Reply:
Thanks @Tlahey for this explanation.
Since @thibault-mahe is on vacation this week and is replacing him during his absence, this PR is open and your explanations seem to be correct. I will therefore take the decision to merge this PR. 
@thibault-mahe, don't hesitate to come back to it if you need to make a modification to these changes via a patch when you come back. 
Thanks!